app/questionnaire/router.py

```diff
@@ -262,7 +262,6 @@ def get_return_to_location_url(
                 return_to_block_id=return_to_block_id,
                 location=location,
                 routing_path=routing_path,
-                is_for_previous=is_for_previous,
```

Comment:
How come this was removed?

Reply:
Not needed anymore, as I took the call to `get_url_for_inaccessible_location` out of the calculated summary function and called it in the parent function instead when the section isn't complete (and that was the only one that needed the parameter)